.github/workflows/ci.yml

```diff
@@ -151,6 +151,10 @@ jobs:
         export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/graphscope/lib:/opt/graphscope/lib64
         source ~/.bashrc
 
+        # install zetcd
+        go get github.com/etcd-io/zetcd/cmd/zetcd
+        cp $(go env GOPATH)/bin/zetcd interactive_engine/bin/
```

Comment:
leave it and use it in its original path? why do we need to move its binary?

Reply:
we just build the `zetcd` binary and no need to install go in docker.